assets/app/scripts/controllers/attachPersistentVolumeClaim.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:CreateRouteController
+ * @description
+ * # CreateRouteController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('AttachPersistentVolumeClaimController', function ($filter, $routeParams, $scope, $window, StorageService, DataService, Navigate, ProjectsService) {
```

Comment:
reeeeally long controller name, I would be ok with using PVC here instead, honestly i'd be ok with PVC in the routes as well like @spadgett suggested